.travis.yml

```diff
@@ -12,12 +19,6 @@ env:
         - DISTRIB="conda" PYTHON_VERSION="3.4"
 
 before_install:
-    - if [ "$DISTRIB" == "ubuntu" ]; then
-         sudo apt-get update -qq;
-         sudo apt-get install -qq python-numpy python-matplotlib
-         python-setuptools;
-      fi
-
```

Comment:
> Just curious, did you have to use sudo=false ? Does that mean that
> python-numpy and al. are installed for all other distribs including the
> conda ones ?

Yes, this is really the right way of doing it. Thanks a lot.

We can't do this in nilearn so far, because neurodebian has not been
added to the whitelist of authorized source.list